src/ashwini/assignment14/Shop.java

```diff
@@ -0,0 +1,182 @@
+package ashwini.assignment14;
+
+import java.util.Scanner;
+import java.lang.String;
+
+/* 1. Create a system (Shop) which maintains each item's quantity.
+
+a. Initial stocks of Maggie: 50 Maggie packets
+b. Initial stocks of Dosa : 43 packets
+c. Initial stocks of pouches : 39 oil packets
+d. Initial stocks of panipuri : 43 packets
+e. Initial stocks of masala’s : 73 packets.
+
+Expectation:
+1) Only one Customer came for shopping, he picked different items with different quantities, print "running out of stock" message if a particular item is sold out otherwise deduct requested quantity from total quantity. Partial selling is allowed.
+2) Create a method which shows only those item which is "Out of stock".
+3) Create a method which shows only those item which is "Available in stock" & respective quantities.NOTE : Please create separate branches for assignment 6 and 7.ś
+*/
+
+public class Shop {
+	int maggieQ;
+	int dosaQ;
+	int oil_packetsQ;
+	int panipuriQ;
+	int masalaQ;
+	String allowed;
+	
+	void setInitialQuantity() {
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Enter the initial stock quantity for maggie: ");
+		maggieQ = scanner.nextInt();
+		System.out.println("Enter the initial stock quantity for dosa: ");
+		dosaQ = scanner.nextInt();
+		System.out.println("Enter the initial stock quantity for oil packets: ");
+		oil_packetsQ = scanner.nextInt();
+		System.out.println("Enter the initial stock quantity for panipuri: ");
+		panipuriQ = scanner.nextInt();
+		System.out.println("Enter the initial stock quantity for masala: ");
+		masalaQ = scanner.nextInt();
+	}
+		
+	void requestedQuantity() {
```

Comment:
We should not ask, user to give inputs after calling methods, 
We should ask user to give input while calling methods, and we take input once only from main method not in every method